### ISSUE TITLE
How to change ringtone on an Android?

### ISSUE THREAD

**Issue by sl5net** [vector-im/element-android#3552]
I want change ringtone on an Android.
i want a subtle soft tone (or no ring) when I make a call.

**Comment by sl5net** [comment]
Matrix Element ringing sound (if u call somebody) is terribly loud,useless, not adjustable, headache

Often misunderstood: it's not about the tone when someone calls, but about the tone when someone else is called (in other words, when you / I call someone else). Matrix Element ringing sound is terribly loud, and not adjustable, makes a headache https://youtu.be/OOulxYMrAPQ

**Comment by fockr** [comment]
I agree. An option to change or disable the ringing sound would be greatly appreciated!

**Comment by jpggithub** [comment]
Also related to
- #2915
- #2936

On my F-droid version (1.4.4), I can change the call ring tone but all the alternatives proposed are notification sounds (that os very short sounds).
Android 11(lineage 18.1) 🤨
